### PR TITLE
Triple citrus grows on trees

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl:
add: Triple citrus can now be grown directly on trees.
/:cl:

Botany power creep nerf now.

fruit
seeds
tree

### CODE REVIEW

File: code/modules/hydroponics/grown/citrus.dm [hunk 1]
```diff
@@ -149,3 +149,28 @@
...
+	potency = 15
+	icon_grow = "lime-grow"
+	icon_dead = "lime-dead"
+	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
+	reagents_add = list("vitamin" = 0.04, "nutriment" = 0.05)
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/citrus/triple
+	seed = /obj/item/seeds/triple
+	name = "Triple citrus"
+	desc = "As sour as a lime, as thick skined as a lemon, and as annoying as an orange."
```
Comment (Iamgoofball) [inline code/modules/hydroponics/grown/citrus.dm]: # STOP

Comment (optimumtact) [inline code/modules/hydroponics/grown/citrus.dm]: what he said

Comment (swindly) [inline code/modules/hydroponics/grown/citrus.dm]: thick-skinned

Comment (Supermichael777) [inline code/modules/hydroponics/grown/citrus.dm]: NO

Comment (Supermichael777) [inline code/modules/hydroponics/grown/citrus.dm]: **UNACEPTABLE**

Comment (Iamgoofball) [inline code/modules/hydroponics/grown/citrus.dm]: f3p0frt
flat 800x800 075 f 2

**Comment by ma44** [comment]
YESSSSSSSSSS

File: code/modules/hydroponics/grown/citrus.dm (hunk 1 above)
Comment (Supermichael777) [inline code/modules/hydroponics/grown/citrus.dm]: i dont know what kind of crappy ornamental lemons your geting but the skin is about 1 cm thick

Comment (Supermichael777) [inline code/modules/hydroponics/grown/citrus.dm]: proof

**Comment by TechnoAlchemisto** [comment]
the seed packet and the fruit do not match!!!!

File: code/modules/hydroponics/grown/citrus.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/hydroponics/grown/citrus.dm]: You spelt it wrong you dimwit

File: code/modules/hydroponics/grown/citrus.dm [hunk 2]
```diff
@@ -149,3 +149,28 @@
 			update_mob()
 			explosion(src.loc,-1,-1,2, flame_range = 5)
 			qdel(src)
+
+// Triple citrus
+/obj/item/seeds/triple
+	name = "pack of triple citrus seeds"
+	desc = "Eat your heart out Frankinstein."
```
Comment (swindly) [inline code/modules/hydroponics/grown/citrus.dm]: Frankenstein

Comment (Iamgoofball) [inline code/modules/hydroponics/grown/citrus.dm]: *Frankenstain

**Comment by Tacolizard** [comment]
@ma44 do you comment that on every botany related pr

**Comment by ShadowDeath6** [comment]
why don't the fruit and the seed packet match

**Comment by Supermichael777** [comment]
because about an hour of time separated them

**Comment by ExcessiveUseOfCobblestone** [comment]
do these not have OJ, Lemon Juice, and Lime Juice in them?

**Comment by ShadowDeath6** [comment]
Probably needs to set something for when you juice it

File: code/modules/hydroponics/grown/citrus.dm [hunk 3]
```diff
@@ -149,3 +149,28 @@
...
+	plantname = "Triple citrus Tree"
+	product = /obj/item/weapon/reagent_containers/food/snacks/grown/citrus/triple
+	lifespan = 55
+	endurance = 50
+	yield = 4
+	potency = 15
+	icon_grow = "lime-grow"
+	icon_dead = "lime-dead"
+	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
+	reagents_add = list("vitamin" = 0.04, "nutriment" = 0.05)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/hydroponics/grown/citrus.dm]: I believe you can add the juices here.

**Comment by ExcessiveUseOfCobblestone** [comment]
you could do it under `reagents_add`

**Comment by optimumtact** [comment]
I do not understand why you would take a goal item in botany and make it available roundstart.

It just reduces what little gameplay botany actually has.

**Pull request closed without merging**